js/views/storeWizardVw.js

```diff
@@ -148,12 +150,19 @@ module.exports = baseModal.extend({
   },
 
   renderModerator: function(moderator){
+    //make sure this moderator is not a duplicate
```

Comment:
I guess this is ok for v1. A better approach is to not allow the duplicates in your data layer, rather than managing a "rendered" data set that is distinct from data. This should be done in the model layer. One way would be to make the guids a nested collection (easier in v2, with code in place to support nesting) in which case you would have no-dupes for free. Another way is to override set(). And, yet another way is to put a validation on the model that kicks back an error if you attempt to add a duplicate to the list.

Anyhow, since our model layer in general in v1 is less than ideal, I guess we could live with the view making the tweak.

To back things up a few steps... why are dupes getting in the list anyway?